apps/ledger-live-mobile/src/screens/Settings/Debug/Generators/ImportBridgeStreamData.tsx

```diff
@@ -11,25 +12,52 @@ type Props = {
 };
 export default function ImportBridgeStreamData({ title, dataStr }: Props) {
   const navigation = useNavigation();
-  return (
-    <SettingsRow
-      title={title}
-      desc={dataStr ? "Import string detected" : "No import string detected"}
-      iconLeft={<Icons.ImportMedium size={24} color="black" />}
-      onPress={() => {
-        if (!dataStr) return;
 
-        const data = JSON.parse(
-          Buffer.from(dataStr, "base64").toString("utf8"),
-        );
+  const importString = useCallback(
+    (dataStr: Parameters<typeof Buffer.from>[0]) => {
+      try {
+        const utf8Str = Buffer.from(dataStr, "base64").toString("utf8");
+        const data = JSON.parse(utf8Str);
 
         navigation.navigate(NavigatorName.ImportAccounts, {
           screen: ScreenName.ScanAccounts,
           params: {
             data,
           },
         });
-      }}
+      } catch (e) {
+        console.error(e);
+      }
+    },
+    [navigation],
+  );
+
+  const importFromClipboard = useCallback(() => {
+    Clipboard.getString().then(str => {
+      importString(
+        str.startsWith("BRIDGESTREAM_DATA=")
+          ? str.replace("BRIDGESTREAM_DATA=", "")
+          : str,
+      );
+    });
+  }, [importString]);
+
+  const handlePress = useCallback(() => {
+    if (!dataStr) importFromClipboard();
+    importString(dataStr);
+  }, [dataStr, importFromClipboard, importString]);
+
+  return (
+    <SettingsRow
+      title={title}
+      desc={
+        dataStr
+          ? "Import string detected. Press to import it. Long press to import the string from the clipboard."
+          : "No import string detected. Press to import the string from the clipboard."
+      }
+      iconLeft={<Icons.ImportMedium size={24} color="black" />}
+      onPress={handlePress}
+      onLongPress={importFromClipboard}
```

Comment:
changes here are a bit unrelated to the goal of this PR but it was very helpful as the only way to import some data from LLD (with lots of accounts) on an emulator is to:
- either set the data in an env -> on build time if the data string is too long it gets sliced.
- copy paste the env data string to set the env dynamically in the env data debug screen -> it seems like the input field there has a length limit.

So here we get the data directly from the clipboard and even if it's very long it seems to work fine.